app/HutchAgent/Services/WorkflowTriggerService.cs

```diff
@@ -0,0 +1,64 @@
+using System.Diagnostics;
+using System.Text;
+using HutchAgent.Models;
+
+namespace HutchAgent.Services;
+
+public class WorkflowTriggerService
+{
+  /// <summary>
+  /// Install and run WfExS given 
+  /// </summary>
+  /// <exception cref="Exception"></exception>
+  public void TriggerWfexs(){
+    
+    const string cmd = "bash";
+    const string activateVenv = "source " + WorkflowTriggerOptions.VirtualEnvironmentPath;
+    
+    // Commands to install WfExS and execute a workflow
+    // given a path to the local config file and a path to the stage file of a workflow
+    var commands = new List<string>()
+    {
+      "apt-get install python3-pip",
+      "yes | ./full-installer.bash",
+      "yes | ./singularity-local-installer.bash",
+      $"./WfExS-backend.py  -L {WorkflowTriggerOptions.LocalConfigPath} execute -W {WorkflowTriggerOptions.StageFilePath}"
+    };
+    
+    var processStartInfo = new ProcessStartInfo
+    {
+      RedirectStandardOutput = true,
+      RedirectStandardInput = true,
+      RedirectStandardError = true,
+      UseShellExecute = false,
+      CreateNoWindow = true,
```

Comment:
Does this cause a new terminal window to spawn? If so we maybe don't want this to be true in prod when it's running on some VM. Could we make this configurable in an appsettings.json?

Reply:
CreateNoWindow is set to true so it doesn't bring up a window when it runs the process

Reply:
Ahh, it would help if I could read parameter names